src/main/webapp/styles/appointment.css

```diff
@@ -0,0 +1,185 @@
+#appointments {
+    font-family: Arial, Helvetica, sans-serif;
+    border-collapse: collapse;
+    width: 100%;
+}
+
+#appointments td, #appointments th {
+    border: 1px solid #ddd;
+    padding: 8px;
+    text-align: center;
+}
+
+#appointments th {
+    padding-top: 12px;
+    padding-bottom: 12px;
+    text-align: left;
+    background-color: #e6f3ff;
+    color: black;
+    text-align: center;
+    position: sticky;
+    top: 0;
+    z-index: 2
+}
+#slots {
+    text-align: center;
+    width: 50%;
+    padding: 10px;
+    font-size: 20px;
+    border-radius: 2px;
+}
+
+#options {
+    border-radius: 45%;
+    width: 110px;
+    height: 100px;
+    outline: 0;
+    font-size: 15px;
+    background-color: #e6f3ff;
+}
+
+.blueAlternateButton {
```

Comment:
My bad here mate. Could we please remove `blueAlternateButton` and `pinkAlternateButton` because I don't think they're in use anymore

Reply:
I'm also seeing duplicate CSS in `Appointments.css` which is confusing me

Reply:
I've seen on slack we're removing this file! Could we delete it here then please and remove `blueAlternateButton` and `pinkAlternateButton` from `main.css`.